s3-proxy/s3-proxy/src/main/java/com/woowacourse/s3proxy/config/StorageConfiguration.java

```diff
@@ -0,0 +1,25 @@
+package com.woowacourse.s3proxy.config;
+
+import com.amazonaws.auth.InstanceProfileCredentialsProvider;
+import com.amazonaws.regions.Regions;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import com.woowacourse.s3proxy.common.fileValidator.FileValidator;
+import com.woowacourse.s3proxy.common.fileValidator.ImageFileValidator;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+
+@Profile("!test")
+@Configuration
+public class StorageConfiguration {
+
+    @Bean
+    public AmazonS3 amazonS3() {
+        return AmazonS3ClientBuilder
+            .standard()
+            .withCredentials(InstanceProfileCredentialsProvider.getInstance())
+            .withRegion(Regions.AP_NORTHEAST_2)
+            .build();
+    }
+}
```

Comment:
```suggestion
public class StorageConfiguration {
    private static final String SEOUL = "Regions.AP_NORTHEAST_2";

    @Bean
    public AmazonS3 amazonS3() {
        return AmazonS3ClientBuilder
            .standard()
            .withCredentials(InstanceProfileCredentialsProvider.getInstance())
            .withRegion(SEOUL)
            .build();
    }
}
```

해당 리전이 어디인지 상수명으로 명시해주면 좋을 거 같아요!
그리고 리전이 다른 곳으로 변경될 수도 있으니까 상수로 빼는 게 어떨까요?!